docs/content/en/docs/Examples/single-node.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Single Node k3s cluster"
+linkTitle: "Single node k3s cluster"
+weight: 1
+description: > 
+    This section describe examples on how to deploy Kairos with k3s as a single-node cluster
+---
+
+In the example below we will use a bare metal host to provision a Kairos node in the local network with K3s.
+
+## Installation
+
+Use the [provider-kairos](https://github.com/kairos-io/provider-kairos) artifacts which contains `k3s`.
+
+Follow the [Installation](/docs/installation) documentation, and use the following cloud config file with Kairos:
+
+```yaml
+#cloud-config
+
+hostname: metal-{{ trunc 4 .MachineID }}
+users:
+- name: kairos
+  # Change to your pass here
+  passwd: kairos
+  ssh_authorized_keys:
+  # Add your github user here!
+  - github:mudler
+
+k3s:
+  enabled: true
+  args:
+  - --disable=traefik,servicelb
+
+# Additional manifests that are applied by k3s on boot
```

Comment:
This part has nothing to do with the single node cluster setup right? I think we already demonstrated this metallb setup here: https://github.com/kairos-io/kairos/pull/618/files#diff-e23d6395ab8483864c81358c557904814cd5a78f16193e2b086363638a188fde